packages/commercetools/helpers/src/index.ts

```diff
@@ -28,6 +30,23 @@ export const getProductVariants = (products: ProductVariant[], options: any = {}
   return products
 }
 
+export const getProductAttributes = (product: ProductVariant, filterByAttributeName?: Array<string>): any => {
+  return (product ? formatAttributeList(product.attributeList) : [])
+  .map(a => ({
+    name: a.name,
```

Comment:
What do you think about this shortcut?
```TS
.map(a => ({
  ...a,
  label: a.label || a.value
}))
```

Reply:
this code willl copy the unnormalized values like 'stringValue'  so it won't work as intended